internal/core/src/common/QueryInfo.h

```diff
@@ -43,6 +43,8 @@ struct SearchInfo {
     bool materialized_view_involved = false;
     bool iterative_filter_execution = false;
     std::optional<SearchIteratorV2Info> iterator_v2_info_ = std::nullopt;
+    std::optional<std::string> json_path_;
+    std::optional<milvus::DataType> json_cast_type_;
```

Comment:
IMHO, it's better to use json_type. there's no cast codes here.

Reply:
changed